app/controllers/deploys_controller.rb

```diff
@@ -21,18 +21,18 @@ def index
     end
   end
 
-  def active_count
-    render json: { deploy_count: active_deploy_scope.count }
-  end
-
   def active
     @deploys = active_deploy_scope
     render partial: 'shared/deploys_table', layout: false if params[:partial]
   end
 
+  def active_count
+    render json: Deploy.active.count
```

Comment:
Either we get the live count here (0-1s race condition) or we don't do the
http call (0-10s race condition) ...

On Tue, May 17, 2016 at 9:57 AM, Shane Hender <notifications@github.com>
wrote:

> In app/controllers/deploys_controller.rb
> <https://github.com/zendesk/samson/pull/985#discussion_r63561328>:
>
> >    def active
> >      @deploys = active_deploy_scope
> >      render partial: 'shared/deploys_table', layout: false if params[:partial]
> >    end
> >
> > +  def active_count
> > +    render json: Deploy.active.count
>
> Not want to use the cache here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/985/files/7139a07ce7500a1ebd185bf9c328eb1cb443dffb#r63561328>
>